docs/contribute/demo-server-setup/OVH-Server-Setup.md

```diff
@@ -0,0 +1,81 @@
+3. Press Save
+
+It should then look like the following picture:
+![InstallationStep2](images/ServerInstallation2.png)
+
+### Step 3: Options
+On the options screen it is important to check the checkbox next to "Use the distribution kernel", beause OVH provides a custom kernel by default, which caused problems in the past in our project.
+![InstallationStep3](images/ServerInstallation3.png)
+
+### Step 4: Confirmation
+The confirmation screen shows the selected template and the language. By pressing the confirm button all data on the current server are wiped and the server is newly installed.
+
+## Server Configuration
+By default it is possible to connect to the server through SSH using a username and password. To make access more secure we only want to allow access with a whitelisted SSH key.
+
+### Creating the SSH key
+Use PuttyGen (or another tool) to create an SSH key. 
+
+Export the generated key as an OpenSSH key (in PuttyGen: Conversions > Export OpenSSH key) and save it as a *.pem file.
+
+Copy the public key and save it in another *.pem file.
+
+Lastly save the private key as *.ppk file as well (in PuttyGen: Button "Save private key"), so that you can use it to log in to the server with Putty.
+
+### Registering the public key
+Connect to the server using SSH (with username and password).
+
+```Bash
+cd ~/.ssh
+vi  authorized_keys2
+```
+add the public key to the end of the file and save it.
+
+Test that login with the public key works by opening another OpenSSH session where you use the private key as authorization.
+
+### Disabling password login
+When login with the private key works, you can disable the password login.
+
+```Bash
+cd /etc/ssh
+vi sshd_config
+```
+
+Disable password authentication by setting `PasswordAuthentication no`.
+
+To ensure that our changes are accepted the ssh service needs to be restarted:
+```Bash
+systemctl restart ssh
+systemctl status ssh
+```
+The second command is to verify that the service was correctly restarted.
+
+An additional test is that when you try to connect with a new OpenSSH session using username and password, the following error message should appear:
+"No supported authentication methods available (server sent: publickey)"
+
+### Updating public key on Circle CI
+Circle CI needs the SSH public key to deploy the server after changes were committed. This is documented on the [Circle CI page](../circle-ci.md#Deploy-key-pair).
+
+### Updating Scenarioo infrastructure
+Scenarioo Infrastructure needs to know the IP address of the new server, so that it can connect and update it.
+
+Set the IP address of the new Server under "mainserver" in [scenarioo-infrastructure\hosts\hosts_demoserver_ovh](https://github.com/scenarioo/scenarioo-infrastructure/blob/master/hosts/hosts_demoserver_ovh).
+
+## Update DNS lookup address
+Finally, the DNS entry needs to be updated with the IP address of the new server, so that http://demo.scenarioo.org points to the new server.
```

Comment:
@mi-we: @bruderol manages the update of the DNS entry. I was unsure whether this should be mentioned or not, so I left it out.

Reply:
@mi-we: As far as I understand it, the Ansible playbook is executed on CircleCI and there is an 'install ansible' step in the CircleCI config.yml in scenarioo-infrastructure.
We didn't have to install anything by hand on the demo server, with the exception of the private key and the configuration changes to log in.

Reply:
> @mi-we: @bruderol manages the update of the DNS entry. I was unsure whether this should be mentioned or not, so I left it out.

Ah ok, I would shortly mention it in the text so that we know who to contact (similar thing as with the OSS licences for IntelliJ)

Reply:
> @mi-we: As far as I understand it, the Ansible playbook is executed on CircleCI and there is an 'install ansible' step in the CircleCI config.yml in scenarioo-infrastructure.
> We didn't have to install anything by hand on the demo server, with the exception of the private key and the configuration changes to log in.

Oh, right, there it is: https://github.com/scenarioo/scenarioo-infrastructure/blob/a21850a4005abd787c3697ab7e483a486871bfba/.circleci/config.yml#L15

Yeah in this case the information is already where it should be. Ok for me